src/screens/simulation/Simulation.js

```diff
@@ -220,6 +216,20 @@ const Simulation = () => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [JSON.stringify(parameters)]);
 
+    const handleImport = ({ version, data }) => {
+        if (version === 1) {
+            // There's no events, everything is in "data" on v1
+            setParameters(data);
```

Comment:
I'm not sure about this ...
In V1, data was a list of timeframes (it was broken with the introduction of the event tab)

Reply:
I would prefer to display an error message than import wrong data

Reply:
I've tested and it's works, what do you mean exactly?